acceptance/indy_acceptance/indy-tests/test_scenarios/test_scenario_03.py

```diff
@@ -0,0 +1,185 @@
+import json
+import sys
+import logging
+import os
+import asyncio
+import shutil
+import time
+from indy import pool, signus, wallet
+from indy.error import IndyError
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from utils.constant import Colors, Constant
+from utils.report import TestReport
+
+
+class MyVars:
+    begin_time = 0
+    pool_handle = 0
+    wallet_handle = 0
+    pool_name = "pool_genesis_test3"
+    wallet_name = "test_wallet3"
+    debug = False
+    test_report = TestReport("Test_scenario_03_Check_Connection")
+    test_results = {"Step 1": False, "Step 2": False, "Step 3": False, "Step 4": False,
+                    "Step 5": False, "Step 6": False, "Step 7": False, "Step 8": False}
+
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def test_precondition():
+    print(Colors.HEADER + "\n\tCheck if the wallet and pool for this test already exist and delete them...\n" + Colors.ENDC)
+
+    if os.path.exists(Constant.work_dir + "wallet/" + MyVars.wallet_name):
+        try:
+            shutil.rmtree(Constant.work_dir + "wallet/" + MyVars.wallet_name)
+        except IOError as E:
+            print(Colors.FAIL + str(E) + Colors.ENDC)
+
+    if os.path.exists(Constant.work_dir + "pool/" + MyVars.pool_name):
+        try:
+            shutil.rmtree(Constant.work_dir + "pool/" + MyVars.pool_name)
+        except IOError as E:
+            print(Colors.FAIL + str(E) + Colors.ENDC)
+
+
+async def test_scenario_03_check_connection():
+    logger.info("Test scenario 3 -> started")
+
+    seed_steward01 = "000000000000000000000000Steward1"
+    pool_config = json.dumps({"genesis_txn": str(Constant.pool_genesis_txn_file)})
+
+    # 1. Create pool ledger
+    print(Colors.HEADER + "\n\t1.  Create pool ledger\n" + Colors.ENDC)
```

Comment:
The same pattern/template is used for each test case (print/try/except).
Can we create an utility method and use this instead of boilerplate code?
I think it will increase test readability and maintenance a lot.

Reply:
I agree. Khoi, please make it so